app/src/main/java/com/whereuat/whereu/ContactRetriever.java

```diff
@@ -0,0 +1,69 @@
+package com.whereuat.whereu;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.ContactsContract;
+import android.content.Context;
+
+/**
+ * Created by whites5 on 3/22/16.
+ */
+public class ContactRetriever {
+    private static Context mContext;
+    private static String phone_num;
```

Comment:
private member variables should start with m and be camel-case (not snake-case), like `mContext` or `mPhoneNum`